src/components/Button.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// eslint-disable-next-line react/prefer-stateless-function
+const Button = ({ color, wide, btnName }) => {
+  const styleColor = ['/', 'X', '-', '+', '='].includes(btnName)
+    ? color[0]
+    : color[1];
+  const widthSize = btnName === '0' ? 345 : wide;
+  return (
+    <div
+      style={{ width: widthSize, backgroundColor: styleColor }}
+      className="calc-btn"
+    >
+      {btnName}
+    </div>
```

Comment:
- [ ] This component is going to be a real button, there is no need to use a div tag here. Use the button tag.